operations/resourcecategories.md

```diff
@@ -0,0 +1,129 @@
+# Resource categories
+
+## Get all resource categories
+
+Returns all categories of resources. This operation uses [Pagination](../guidelines/pagination.md).
+
+### Request
+
+`[PlatformAddress]/api/connector/v1/resourceCategories/getAll`
+
+```javascript
+{
+    "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+    "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+    "Client": "Sample Client 1.0.0",
+    "ResourceCategoryIds": [
```

Comment:
Should we include an example for "EnterpriseIds" here? It's an optional field, and most users will not need to specify it, so I'm not sure.